src/sentry/api/endpoints/catchall.py

```diff
@@ -2,14 +2,15 @@
 from django.views.decorators.csrf import csrf_exempt
 from rest_framework.request import Request
 
-from sentry.api.base import Endpoint, all_silo_endpoint
+from sentry.api.base import Endpoint, all_silo_endpoint, allow_cors_options
 
 
 @all_silo_endpoint
 class CatchallEndpoint(Endpoint):
     permission_classes = ()
 
     @csrf_exempt
+    @allow_cors_options
```

Comment:
I think this "fixed" the mypy errors list because these are supposed to be method decorators via `@method_decorator(...)` and this is not actually working properly 🤔

Reply:
that or `allow_cors_options` is untyped so it hides an error behind `Any`

Reply:
allow_cors_options is currently untyped :cry: However, the decorator is doing its job of adding the necessary headers.